WordPressKit/AccountSettingsRemote.swift

```diff
@@ -90,6 +90,26 @@ public class AccountSettingsRemote: ServiceRemoteWordPressComREST {
                                  })
     }
 
+    /// Validate the current user's username
+    ///
+    /// - Parameters:
+    ///   - username: The new username
+    ///   - success: block for success
+    ///   - failure: block for failure
+    public func validateUsername(to username: String, success: @escaping () -> Void, failure: @escaping (Error) -> Void) {
+        let endpoint = "me/username/validate/\(username)"
+        let path = self.path(forEndpoint: endpoint, withVersion: ._1_1)
+        
+        wordPressComRestApi.GET(path,
+                                parameters: nil,
+                                success: { responseObject, httpResponse in
+                                    success()
```

Comment:
I'm curious why nothing is done with the responseObject.  The results contain a list of available actions that include: `new`, `none`, `rename_discard` and `rename_keep` which determine the behavior a username change has on the user's blog.  Is the expectation always that a change will be made with `none`?  

Maybe add a comment here to clarify what the intent is?

Reply:
Testing the API response I can see only `new` and `none` as options:
```
{
    "success": true,
    "allowed_actions": {
        "new": "Yes, create a new blog to match my new user name",
        "none": "No, don't create a matching blog address."
    }
}
```
In Calypso too. I left it _blank_ because the `changeUsername` method set the action as `none`. I'll leave a comment saying that the block should be changed if the `allowed_actions` is required.

Reply:
I looked a little at the endpoint code earlier but I don't remember the circumstances that would return all four values. Fwiw, this is what I see from my `aerychtest` account.
<img width="1020" alt="Screen Shot 2019-07-25 at 12 52 06 PM" src="https://user-images.githubusercontent.com/1435271/61896759-38787500-aedb-11e9-9da0-dd60a9e69410.png">